sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -434,24 +438,31 @@ object ScalaNativePluginInternal {
       (nativeExternalDependencies.value.toSet --
         nativeAvailableDependencies.value.toSet).toList.sorted
     },
-    nativeAvailableDependencies := ResourceScope { implicit scope =>
-      val forceCompile = compile.value
-
-      val globals = fullClasspath.value
-        .collect { case p if p.data.exists => p.data }
-        .flatMap(p => tools.LinkerPath(VirtualDirectory.real(p)).globals.toSeq)
-
-      globals.map(_.show).sorted
-    },
-    nativeExternalDependencies := ResourceScope { implicit scope =>
+    nativeAvailableDependencies := Def
+      .task {
+        val fcp = fullClasspath.value
```

Comment:
This is now an error in sbt.

It's to guard us from the bogus sbt evaluation model. For example:

```
val foo =
  if(false) task1.value
  else task2.value
```

both tasks will be evaluated.